nuc_morph_analysis/analyses/volume/FigS10_workflow.py

```diff
@@ -0,0 +1,268 @@
+from nuc_morph_analysis.analyses.volume import filter_out_dips
+from nuc_morph_analysis.analyses.neighbor_of_X.misc_neighbor_helper_functions import get_a_cells_neighbors_as_track_id_list
+
+from nuc_morph_analysis.lib.visualization.plotting_tools import get_plot_labels_for_metric
+from nuc_morph_analysis.lib.visualization import plotting_tools
+from nuc_morph_analysis.analyses.volume_variation import plot_features
+
+#%%
+# now load data with growth outliers
+df_outliers = load_dataset_with_features('all_baseline', remove_growth_outliers=False)
+# and without outliers
+df = load_dataset_with_features('all_baseline', remove_growth_outliers=True)
+#%%
+# apply minimal filtering
+df_outliers = filter_data.all_timepoints_full_tracks(df_outliers)
+df = filter_data.all_timepoints_full_tracks(df)
+#%%
+# update plotting parameters
+fs,fw,fh = update_plotting_params()
+save_dir = Path(__file__).parent / 'figures' / 'volume_dip_figures'
+# %%
+# S10 panel A, left
+# choose one track and its neighbors (at a given time) to plot over time
+MAIN_TRACK_ID = 75725
+TIMEPOINT = 239
+track_id_list = get_a_cells_neighbors_as_track_id_list(df_outliers,MAIN_TRACK_ID,TIMEPOINT)
+fig,axlist = plot_neighbors_volume_over_time(df_outliers,track_id_list)
+
+save_name = f"S10_A_left-immediate_neighbors_of_main_track_{MAIN_TRACK_ID}"
+save_path = str(save_dir / save_name)
+for ext in ['.png','.pdf']:
+    save_and_show_plot(save_path,ext,fig,transparent=False,keep_open=True)
+plt.show()
+
+#%%
+# S10 panel A, middle
+fig,axlist = plot_tracks_aligned_at_volume_drop_onset(df_outliers,track_id_list,MAIN_TRACK_ID,TIMEPOINT)
+
+save_name = f"S10_A_middle-dip_shape_{MAIN_TRACK_ID}"
+save_path = str(save_dir / save_name)
+for ext in ['.png','.pdf']:
+    save_and_show_plot(save_path,ext,fig,transparent=False,keep_open=True)
+plt.show()
+
+#%%
+# S10 panel A, right
+# view data in timelapse feature explorer (TFE) with the following link 
+# TODO: update data path to be the final TFE url
+datapath = "https%3A%2F%2Fdev-aics-dtp-001.int.allencell.org%2Fassay-dev%2Fusers%2FFrick%2FPythonProjects%2Frepos%2Flocal_storage%2Ftimelapse_feature_explorer_datasets%2FTFE_new%2Fexploratory_dataset%2Fsmall%2Fmanifest.json"
+url = f"https://timelapse.allencell.org/viewer?dataset={datapath}&feature=change_in_volume_in_25_minute_window&t=239&filters=growth_outlier_filter%3A%3A3%2Cbaseline_colonies_dataset_filter%3A%3A3%2Cfullinterphase_dataset_filter%3A%3A3%2Clineageannotated_dataset_filter%3A%3A3%2Cvolume_jumps_right_magnitude_mask%3A%3A43.916%3A254.426&range=-200%2C200&color=matplotlib-purple_orange&palette-key=adobe&bg-sat=100&bg-brightness=100&fg-alpha=100&outlier-color=c0c0c0&outlier-mode=1&filter-color=dddddd&filter-mode=1&tab=scatter_plot&scalebar=1&timestamp=1&path=1&keep-range=1&scatter-range=all&scatter-x=scatterplot_time&scatter-y=volume"
+print(url)
+# save url as text file in figure folder
+with open(save_dir / 'S10_A_right_url.txt','w') as f:
+    f.write(url)
```

Comment:
This seems nice but I dont think it is necessary. We dont do this for the other figures made with TFE

Reply:
We also have a TFE links table in the docs section!! We can place the link there!